src/lib/pre_diff_info/intf.ml

```diff
@@ -0,0 +1,119 @@
+open Core
+open Protocols.Coda_pow
+
+module type Inputs = sig
```

Comment:
if you can get away with it, depend on Coda_base and don't functor over staged ledger (et al).

Reply:
If this is ripping out a piece of staged-ledger, don't worry about changing the functor, but this file should probably live int he staged_ledger library still in that case.

Reply:
Ok cool, I will put pre_diff_info into staged_ledger module